components/src/molecules/ParametersTable/NoParameters.tsx

```diff
@@ -29,9 +33,7 @@ export function NoParameters({ t }: NoParametersProps): JSX.Element {
         aria-label="alert"
       />
       <StyledText as="p" fontWeight={TYPOGRAPHY.fontWeightSemiBold}>
-        {t != null
-          ? t('no_parameters')
-          : 'No parameters specified in this protocol'}
+        {t != null ? t(textKey) : 'No parameters specified in this protocol'}
```

Comment:
If the above comment makes sense to you

```suggestion
        {'No parameters specified in this protocol'}
```